raft/raft.go

```diff
@@ -1583,7 +1583,7 @@ func (r *raft) loadState(state pb.HardState) {
 	r.Vote = state.Vote
 }
 
-// pastElectionTimeout returns true iff r.electionElapsed is greater
+// pastElectionTimeout returns true if r.electionElapsed is greater
```

Comment:
perhaps `iff` here is intentional, not a typo. check https://en.wikipedia.org/wiki/If_and_only_if